### PR TITLE
vendor gardener-1.1.0

### PR DESCRIPTION
**Pull request by mvladev**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement developer
`github.com/gardener/gardener` dependency updated to `v1.1.0`
```

### CODE REVIEW

**Review by rfranzke — DISMISSED** [review]

**Review by timebertt — DISMISSED** [review]
/lgtm

File: pkg/controller/worker/genericactuator/actuator.go [hunk 1]
```diff
@@ -161,7 +161,7 @@ func (a *genericActuator) listMachineClassSecrets(ctx context.Context, namespace
 	var (
 		secretList = &corev1.SecretList{}
 		labels     = map[string]string{
-			v1beta1constants.GardenPurpose: GardenPurposeMachineClass,
+			v1beta1constants.GardenerPurpose: GardenPurposeMachineClass,
```
Comment (ialidzhikov) [inline pkg/controller/worker/genericactuator/actuator.go]: This is a behavioural change, currently the machine class secrets are labelled with `garden.sapcloud.io/purpose`, this change tries to list them with `gardener.cloud/purpose`. Is it backwards compatible?
I would suggest to define here the `GardenPurpose` const to the deprecated label key and probably to handle the adaptation with other PR. WDYT?

Comment (mvladev) [reply pkg/controller/worker/genericactuator/actuator.go]: I've updated the `listMachineClassSecrets` implementation to handle this problem.

**Review by rfranzke — APPROVED** [review]

**Review by ialidzhikov — APPROVED** [review]
Thank you!

/lgtm

**Review by timebertt — APPROVED** [review]
/lgtm

**Pull request merged**